packages/dds/tree/src/test/edit-manager/editManger.spec.ts

```diff
@@ -0,0 +1,654 @@
+    }
+
+    public invert(change: TestChangeset): TestChangeset {
+        if (isNonEmptyChange(change)) {
+            return {
+                inputContext: change.outputContext,
+                outputContext: change.inputContext,
+                intentions: change.intentions.map((i) => -i).reverse(),
+            };
+        }
+        return emptyChange;
+    }
+
+    public rebase(change: TestChangeset, over: TestChangeset): TestChangeset {
+        if (isNonEmptyChange(change)) {
+            if (isNonEmptyChange(over)) {
+                return {
+                    inputContext: over.outputContext,
+                    // Note that we mint a new context ID for each rebased operation.
+                    // This means that rebasing some change A over some change B will produce
+                    // a change with a different output context every time.
+                    // This lack of fidelity could make some of the tests fail when they
+                    // should not, but will not make tests pass if they should not.
+                    // If a rebaser implementation needed to leverage this missing fidelity then this gap could
+                    // be addressed by using a more complex encoding to represent contexts.
+                    outputContext: ++this.contextCounter,
+                    intentions: change.intentions,
+                };
+            }
+            return change;
+        }
+        return emptyChange;
+    }
+
+    public rebaseAnchors(anchor: AnchorSet, over: TestChangeset): void {
+        throw new Error("Method not implemented.");
+    }
+
+    public import(change: TestChangeset): TestChangeset {
+        return change;
+    }
+
+    public export(change: TestChangeset): TestChangeset {
+        return change;
+    }
+
+    public mintChangeset(inputContext: number): NonEmptyTestChangeset {
+        return {
+            inputContext,
+            outputContext: ++this.contextCounter,
+            intentions: [++this.intentionCounter],
+        };
+    }
+
+    public checkChangeList(changes: readonly RecursiveReadonly<TestChangeset>[], intentions?: Set<number>): void {
+        const filtered = changes.filter(isNonEmptyChange);
+        const intentionsSeen = new Set<number>();
+        const intentionsExpected = new Set<number>(
+            intentions ??
+            makeArray(this.intentionCounter, (i: number) => i + 1),
+        );
+        let index = 0;
+        for (const change of filtered) {
+            for (const intention of change.intentions) {
+                if (intention > 0) {
+                    // The same intention should never be applied multiple times
+                    assert.strictEqual(intentionsSeen.has(intention), false);
+                    intentionsSeen.add(intention);
+                    // The intention should be part of the expected set for this client
+                    assert.strictEqual(intentionsExpected.has(intention), true);
```

Comment:
The documentation on `strict` seems to imply that `strict.equal` behaves the same as `strict.strictEqual`, so I think you can change shorten all these usages.